util/src/main/java/com/liferay/faces/util/application/ResourceDependencyHandlerFactory.java

```diff
@@ -19,30 +19,17 @@
 
 
 /**
- * @author  Neil Griffin
+ * This class provides the contract for a factory which can create {@link ResourceDependencyHandler}s.
+ *
+ * @author  Kyle Stiemann
  */
-public abstract class ComponentResourceWrapper implements ComponentResource, FacesWrapper<ComponentResource> {
```

Comment:
Github is too smart for its own good. This is an API change: removed `ComponentResourceWrapper`. It could be deprecated instead, but it is very low risk because non-Faces projects are probably not using it.